plugins/org.csstudio.trayicon/src/org/csstudio/trayicon/TrayIcon.java

```diff
@@ -114,7 +106,7 @@ public void handleEvent(Event event) {
 
     private Menu createPopupMenu() {
         // Create a Menu
-        Menu popupMenu = new Menu(window.getShell(), SWT.POP_UP);
+        Menu popupMenu = new Menu(windows[0].getShell(), SWT.POP_UP);
```

Comment:
Does this need better null checking?
`windows` will be set here; are there any use-cases in which `getWorkbenchWindows` could return an empty list?